### DISCUSSION TITLE
Auto-Collapse Section

### DISCUSSION THREAD

**Discussion by numenor-dev** [twilio-labs/paste#2975] [Q&A] [answered]
Hi,

Is there a way to auto-collapse a section based on specific breakpoints? The section I'm wanting to modify on our site currently has a manual button toggle to collapse the section but ideally it would be great for it to auto-collapse on smaller viewports.

**Comment by loreina** [comment, marked as answer]
Hi @numenor-dev, could you share a screenshot for additional context?

**Reply by numenor-dev** [reply]
Hi @loreina,
<img width="340" alt="Screen Shot 2023-01-24 at 1 47 06 PM" src="https://user-images.githubusercontent.com/27752036/214430092-b4796513-298d-4a53-9f1c-8ee329c83c70.png">


Here is a screenshot of the section I'd like to set up as auto-collapse. The chevron icons (button) will manually collapse it, but it's not intuitive so I am hoping there are Twilio components or primitives that would allow this section to auto-collapse when reaching a certain pixel width.

**Reply by gloriliale** [reply]
Hii @numenor-dev! I think the [Grid component's responsiveness](https://paste.twilio.design/components/grid#responsive-columns) would help achieve what you're looking to do. It works with the theme's breakpoints, which for our default theme are 25rem (400px), 64rem (1024px), and 77rem (1232px). Hope that helps!

**Reply by numenor-dev** [reply]
Hi @gloriliale,

Awesome, thanks a ton! I'll give this a shot.
